web/app/js/components/util/MetricUtils.js

```diff
@@ -31,10 +31,18 @@ export const getPodsByDeployment = pods => {
     .reject(p => _.isEmpty(p.deployment) || p.controlPlane)
     .groupBy('deployment')
     .map((componentPods, name) => {
+      _.remove(componentPods, p => {
+        return p.status === "Terminating" || p.status === "Completed";
```

Comment:
In playing around with this locally, I realized that there isn't a "Completed" status, unfortunately. The five [pod phases](https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#pod-phase) are: Pending, Running, Succeeded, Failed, and Unknown. We've also added Terminating for pods that are terminating.

I haven't thought about this too much, but I think we should should categorize pods as follows:

* `p.added && p.status == "Running"` => "good"
* `p.status == "Pending"` => "neutral"
* `p.status == "Failed"` => "bad"
* `p.status == "Terminating" || p.status == "Succeeded" || p.status == "Unknown"` => remove

But I'd also be up for other suggestions of how to categorize and filter.